relay-event-schema/src/protocol/span.rs

```diff
@@ -274,7 +274,8 @@ pub struct SpanData {
     /// Origin Transaction name of the span.
     ///
     /// For INP spans, this is the route name where the interaction occurred.
-    pub transaction: Annotated<String>,
+    #[metastructure(field = "segment.name", legacy_alias = "transaction")]
+    pub segment_name: Annotated<String>,
```

Comment:
Renamed to future key. `transaction` becomes a legacy alias.

Reply:
Should we update the docs too?